gcalcli/argparsers.py

```diff
@@ -108,9 +108,10 @@ def __call__(self, parser, namespace, value, option_string=None):
 
 
 def validwidth(value):
+    minwidth=30
```

Comment:
This change from 10 to 30 is because 10 was just intended as the min size for one day cell vs whole output? Is there a specific reason for picking 30?

Reply:
It's kind of the minimal width that still makes any sense for `gcalcli agenda --details description`. For 30 it comes out to about a width of 5 chars for the description. But I'm happy to change it to anything else obviously.
```
Mon Jan 27  8:30           Strategy meeting
                     Description:
                     ┌───────┐
                     │ aaaaa │
                     │ aaaaa │
                     │ aaaaa │
                     │ aaaaa │
```

Reply:
Nah I doubt it'll bother anyone, just wanted to understand if it's an intentional behavior change.